contrib/sip_proxy/filters/network/source/conn_manager.h

```diff
@@ -360,6 +337,12 @@ class ConnectionManager : public Network::ReadFilter,
   void sendLocalReply(MessageMetadata& metadata, const DirectResponse& response, bool end_stream);
   void doDeferredTransDestroy(ActiveTrans& trans);
   void resetAllTrans(bool local_reset);
+  absl::string_view localAddress() {
+    auto& local_address =
+        context_.getTransportSocketFactoryContext().localInfo().address()->ip()->addressAsString();
+    ENVOY_LOG(debug, "localAddress {}", local_address);
```

Comment:
Nit: probably should drop this debug, don't think it will be useful post-dev.

Reply:
done. Thanks for your comments.